code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -125,17 +125,6 @@
 	new /obj/item/storage/fancy/candle_box(src)
 	return
 
-/obj/structure/closet/coffin

```

Comment:
why did you remove the coffin

Reply:
its a refactor, I mentioned it in the PR

Reply:
@optimumtact Check crates.dm